client/state/index.js

```diff
@@ -119,13 +119,15 @@ const atEnabled = isDevOrWPCalypso && config.isEnabled( 'automated-transfer' );
 
 export function createReduxStore( initialState = {} ) {
 	const isBrowser = typeof window === 'object';
+	const isAudioSupported = typeof Audio === 'function';
```

Comment:
just a note that this check is different than the one inside the middleware where we test that we're in a window _and_ that `Audio` is a function